main.go

```diff
@@ -113,6 +114,7 @@ func main() {
 				awsManager.CleanVpcInstances(assumedRoleClient)
 				awsManager.CleanEbsSnapshots(assumedRoleClient)
 				awsManager.CleanEbsVolumes(assumedRoleClient)
+				awsv1alpha1.ResetAccountState(account)
```

Comment:
We don't want to do this for each region. We only want to run this if all regions are totally cleaned with no errors.